advanced-cache.php

```diff
@@ -483,10 +485,11 @@ function add_debug_html_to_output( $debug_html ) {
 if ( $batcache->do )
 	$batcache->genlock = wp_cache_add("{$batcache->url_key}_genlock", 1, $batcache->group, 10);
 
-if ( isset( $batcache->cache['time'] ) && // We have cache
+if (
+	isset( $batcache->cache['time'] ) && // We have cache
```

Comment:
nit-picking: I know it's not necessarily part of this PR, but since we are already re-using the `$is_cached` variable elsewhere, can't we use it here as well?

Would it be better if the `$is_cached` was defined as `is_array( $batcache->cache ) && isset( $batcache->cache['time'] )`?

Reply:
Updated. Please take one more look, and let's :shipit:.

Reply:
Thanks for addressing that feedback!